### ISSUE TITLE
Text hidden because filter geometry differs between 18.06.02 & 18.11.18

### ISSUE THREAD

**Issue by CanRau** [mltframework/shotcut#680]
macOS 10.12.6
Shotcut 18.11.18

TL;DR
the former version (18.06.02) saved text geometry property as
```
<property name="geometry">0,9375%/79,8148%:97,8125%x9,62963%</property>
```
whereas the new one uses
```
<property name="geometry">0 431 939 52 1</property>
```
![shotcut-text_filter_geometry_prop_issue-680](https://user-images.githubusercontent.com/5196971/50067388-d4484280-018e-11e9-9bec-4e502d44c2be.gif)

Luckily I could just replace all those lines easily with the new format  

It might not be a big problem anymore and maybe it would've even be solved if I had updated more often?!..
Thought it's nice to file it anyway as it might interest someone 😉 
Thanks for this great tool! I'm still quite new to Shotcut and even pretty unexperienced in video editing in general, touched it only a few times in my life using different apps…

The problem I'm facing came from updating 18.06.02 to 18.11.18.
I made a video in the former version, with text filters containing subs. Then I upgraded, translated the subs in a text editor opening the .mlt and realized the text is not showing up when opening the edited .mlt in new newest version. I downgraded which would bring back the text filters. Upgrading again and investigating more I noticed that the text is shown in the filter properties, just not in the preview and changing the position back and forth solves the issue.

Feel free to close, even uncommented if it's inappropriate and helps save time
Greets from the peruvian jungle

**Comment by CanRau** [comment]
Okay got it, was a little confusing and I didn't know how to find information about it and what the suggested way of solving it is 😉 but I solved it so never mind 👍
